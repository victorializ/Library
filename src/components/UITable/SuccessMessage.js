import React from 'react';
import { Button, Dropdown, Segment } from 'semantic-ui-react';

import { colors } from '../../assets/semantic-colors';
import { constants } from '../../i18n';

const DefaultSuccessMessage = () => 
    <Segment color={colors.primary}>
        {constants.success}
    </Segment>;

export { DefaultSuccessMessage };