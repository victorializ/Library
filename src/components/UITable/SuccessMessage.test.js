import React from 'react';
import { shallow } from 'enzyme';

import { DefaultSuccessMessage } from './SuccessMessage';

describe('SuccessMessage component', () => {
    it('renders correctly', () => {
        expect(shallow(<DefaultSuccessMessage />)).toMatchSnapshot();
    });   
});