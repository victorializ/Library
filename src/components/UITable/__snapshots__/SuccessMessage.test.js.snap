// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessMessage component renders correctly 1`] = `
<Segment
  color="olive"
>
  Success!
</Segment>
`;
